tests/base_test.py

```diff
@@ -13,26 +14,24 @@
 '''
 class ArkoudaTest(unittest.TestCase):
 
-    verbose = bool(os.getenv('ARKOUDA_VERBOSE', 'False'))
-    port = os.getenv('ARKOUDA_SERVER_PORT')
-    server = os.getenv('ARKOUDA_SERVER_HOST')
-
-    port_server_set = port != None and server != None
-    full_stack_mode = bool(os.getenv('ARKOUDA_FULL_STACK_TEST', not port_server_set))
+    verbose = True if os.getenv('ARKOUDA_VERBOSE') == 'True' else False
+    port = int(os.getenv('ARKOUDA_SERVER_PORT', 5555))
+    server = os.getenv('ARKOUDA_SERVER_HOST', 'localhost')
+    full_stack_mode = True if os.getenv('ARKOUDA_FULL_STACK_TEST') == 'True' else False
     
     @classmethod
     def setUpClass(cls):
         '''
-        If in full stack mode, Configures and starts the arkouda_server process
+        If in full stack mode, Configures and starts the arkouda_server process, noop otherwise
         
         :return: None
         :raise: RuntimeError if there is an error in configuring or starting arkouda_server
         '''
         if ArkoudaTest.full_stack_mode:
             print('starting in full stack mode')
             try: 
-                nl = get_arkouda_numlocales()
-                (ArkoudaTest.server, ArkoudaTest.port, ArkoudaTest.ak_server) = start_arkouda_server(nl)
+                nl = int(get_arkouda_numlocales())
```

Comment:
Nit but `get_arkouda_numlocales()` should already return an int

Reply:
Good point, will clean this up.